### DISCUSSION TITLE
first load (initial load) takes too long

### DISCUSSION THREAD

**Discussion by yousef-repzo** [vercel/community#934] [Help] [answered]
sometimes the app take too long to load (initial load) so i opened the network to see the timing (waiting from server to response) it was **22 sec** to bring **50 kB** chunk
even sometimes takes 10sec to bring 30B chunk
![chunk](https://user-images.githubusercontent.com/104502702/192775355-7ae7a279-5e03-4554-84ed-2c41f6a9707a.PNG)

Vite Config
![viteconfig](https://user-images.githubusercontent.com/104502702/192947631-b83d0fa7-085a-444f-ab36-0c6c18b7ac24.PNG)

vercel config
![vercelconfig](https://user-images.githubusercontent.com/104502702/192947683-63356143-6672-4840-af6e-59ce37cb2e67.PNG)

tsconfig
![tsconfig](https://user-images.githubusercontent.com/104502702/192947703-65370451-c590-4d1b-8d50-2e917d850679.PNG)

by the way i faced this issue when i migrate from **create react app** to **vite**
but then i return to CRA and still same issue

**Comment by amyegan** [comment, marked as answer]
Hi @yousef-repzo! Are you able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) and example deployment so I can see the problem for debugging purposes?

**Reply by yousef-repzo** [reply]
Hi @amyegan thank you for responding
i updated the the issue

**Reply by amyegan** [reply]
Sorry for the late response. I think this problem is related to another issue that was also reported last week. A fix is live and confirmed to be working for at least one customer. Are you still experiencing this problem?

**Reply by yousef-repzo** [reply]
@amyegan thanks 
seems to be solved
